gn2/wqflask/views.py

```diff
@@ -300,14 +287,7 @@ def __success__(resp):
                 error_page, __success__)
         except Exception as error:
             return flask.jsonify({"error": str(error)})
```

Comment:
error type missing i.e. `return flask.jsonify({"error": str(error)}), 500`, otherwise we'll assume 200

Reply:
Actually this exception is  unreachable the general exception handler can handle this:
todo: remove